lean4-util.el

```diff
@@ -38,16 +36,16 @@ Try to find an executable named `lean4-executable-name' in variable `exec-path'.
 On succsess, return path to the directory with this executable."
   (let ((root (executable-find lean4-executable-name)))
     (when root
-      (setq lean4-rootdir (f-dirname (f-dirname root))))
+      (setq lean4-rootdir (file-name-directory (directory-file-name root))))
```

Comment:
```suggestion
      (setq lean4-rootdir
            (file-name-directory
             (directory-file-name
              (file-name-directory root)))))
```
There were _two_ calls to `f-dirname`! (I don't know why `f-parent` wasn't used.)

But also: the value should end in a slash since it's going to be used as `default-directory` later. (This was already incorrect before your changes.)

Reply:
@phikal, can you fix this on your branch and push, please?

Reply:
I'm sorry, I didn't realise that these was supposed to be "suggested changes".